.circleci/config.yml

```diff
@@ -213,3 +216,14 @@ workflows:
               only: master
           requires:
           - deploy
+  push-to-app-catalog:
+    jobs:
+      - architect/push-to-app-catalog:
+          name: "package and push chart-operator-chart"
+          app_catalog: "giantswarm-catalog"
+          app_catalog_test: "giantswarm-test-catalog"
+          chart: "chart-operator-chart"
+          # Trigger job on git tag.
+          filters:
+            tags:
+              only: /^v.*/
```

Comment:
You will also need to add push-to-docker: https://github.com/giantswarm/aws-operator/pull/1808/files

There is not yet merged guide we are working on: https://github.com/giantswarm/giantswarm/pull/6656/files

Reply:
@kopiczko Should I add `push-to-docker` as in the same workflow `push-to-catalog`?

Reply:
Yes https://github.com/giantswarm/aws-operator/blob/15592ef325d5287fac2f4a791517da5f91301589/.circleci/config.yml#L233-L253